backend/entities/organization_entity.py

```diff
@@ -95,4 +100,29 @@ def to_model(self) -> Organization:
             youtube=self.youtube,
             heel_life=self.heel_life,
             public=self.public
+        )
+        
+    def to_details_model(self) -> OrganizationDetails:
+        """
+        Converts a `OrganizationEntity` object into a `OrganizationDetails` model object
+        
+        Returns:
+            OrganizationDetails: `OrganizationDetails` object from the entity
+        """
+        return OrganizationDetails(
+            id=self.id,
+            name=self.name,
+            shorthand=self.shorthand,
+            slug=self.slug,
+            logo=self.logo,
+            short_description=self.short_description,
+            long_description=self.long_description,
+            website=self.website,
+            email=self.email,
+            instagram=self.instagram,
+            linked_in=self.linked_in,
+            youtube=self.youtube,
+            heel_life=self.heel_life,
+            public=self.public,
+            events=[event.to_model() for event in self.events]
```

Comment:
Something we'll need to think about: do we actually want `events` to be a part of an `OrganizationDetails` model or should we just have a separate end point for querying the events of a specific org? My concern is looking toward the future when past events outnumber upcoming events: it seems this is getting all historical events when that's unlikely to be what shows in the page or of interest to the user.

Reply:
@KrisJordan this is a good point. We used this convention to prevent using two API calls on the organization details page.

Reply:
This is fine for the time being, but a feature extension we should prioritize next sprint is resolving pagination of events (esp backwards looking) and improving this endpoint so that it only gives forward looking events.